Arooshaa17/courseswork

arooshaa17 opened the pull request:
@raeeschachar

raeeschachar commented on pages/locators.py:
```diff
@@ -39,3 +39,14 @@
+blog = "//a[contains(text(),'Blog')]"
+contact_us = "//a[contains(text(),'Contact Us')]"
+help_c = "//a[contains(text(),'Help Center')]"
+footer_elements_list = [footer_logo, about_link, for_business, terms, privacy, accessibility, blog, contact_us, help_c]
```
empty line at the end of file

raeeschachar replied:
please move this to test instead of locators

raeeschachar commented on pages/home_page.py:
```diff
@@ -16,3 +16,21 @@ def verify_homepage_nav_bar(self, selectors_list):
+        is_present = None
+        for selector in selectors_list:
+            self.element_click(selector)
+            print(self.driver.title)
```
please remove print statement

Commit:
'abc'

My work of April 22

My work of April 22

Commit of April 23

My work of May 4th

My work of May 8
